### ISSUE TITLE
Paned brower: refresh selection when fields are modified

### ISSUE THREAD

**Issue by Jerome-github** [quodlibet/quodlibet#1741]
(Using version 3.2.2 from Debian Jessie.)

In the paned browser, my album list in the rightmost columns displays "date - name", therefore it is sorted by date. If yours displays "name", just transpose the example by modifying the name, not the date.

Select an album from the album list. Select all files and update the date in the tags.

The album is moved in the list because the "date - name" field just changed.

The selection is still at the same place: the Nth element of the list is selected, except it is not the same album anymore. The same tracks/files are still shown/selected, though.

We are in an inconsistent state, here, as another album seems to be selected but the tracks displayed are those of the one we just modified.

The selection in the album column should be refreshed to select the new index of the album we're displaying.

**Comment by lazka** [comment]
(we provide a debian jessie repo if you want to give 3.5.1 a try)

A quick fix would be to remove the selection in this case... we don't update the list atm because we only support replacing the whole list and that is slow when lots of updates happen.

**Comment by Jerome-github** [comment]
I didn't know about the Jessie repo. I don't usually need the very latest version, but I'll probably use it to validate a fix or as a check before submitting a new bug report.

I'm not sure I understand your answer. Maybe I was unclear in the first place. Let me rephrase shorter and hopefully clearer.

Basically, lists are displayed in each pane. When modifying tags results in items in the list being reordered, the list actually is reordered, but the selection remains at the same "position", that is the same index, I guess, when it should "follow" its item, eventually changing index/position accordingly.

**Comment by lazka** [comment]
> Basically, lists are displayed in each pane. When modifying tags results in items in the list being reordered, the list actually is reordered, but the selection remains at the same "position", that is the same index, I guess, when it should "follow" its item, eventually changing index/position accordingly.

ah ok.. another issue. we currently remove all changed entries and then add them back in the right position which is why we "forget" them.
